### DISCUSSION TITLE
Transform docgen output.

### DISCUSSION THREAD

**Discussion by lazenyuk-dmitry** [storybookjs/storybook#24153] [Help]
Hi everyone. I use Vue. I have a next problem. 

I created a component with prop "title" and slot "title". But the storybook can`t handle doc props with identical names. Is there a way to control documentation generation?

https://github.com/vue-styleguidist/vue-styleguidist/tree/dev/packages/vue-docgen-api#options-docgenoptions

I want to transform the slot name "title" to "#title" on docgen output. So that the documentation is displayed correctly.

**Comment by jdassin** [comment]
If you encounter any questions or problems with the Storybook & Vue library, please check out https://github.com/storybook-vue. It's focused on Vue and managed by the knowledgeable @chakAs3, who is also the Storybook maintainer. You can expect a quick and helpful response from him.
